### ISSUE TITLE
ifcDoor.GetPropertySingleValue can not handle duplicate propertysets	

### ISSUE THREAD

**Issue by jojobs** [xBimTeam/XbimWindowsUI#90]
Hi,
I have an IFC file with multiple "Dimensions" propertysets in Doors. Therefore I can only see properties in first set using GetPropertySingleValue function. Is there any work around?

![multipledimesnions](https://user-images.githubusercontent.com/18474892/43698798-2850a5ec-998f-11e8-87df-a70ac72845d1.png)

[Test4-180704_MappedPropKelly3.zip](https://github.com/xBimTeam/XbimWindowsUI/files/2261271/Test4-180704_MappedPropKelly3.zip)

**Comment by martin1cerny** [comment]
`GetPropertySingleValue()` is just a shortcut which makes certain assumptions. Access the data directly to have a complete access.

**Comment by martin1cerny** [comment]
Yes, we assume it is unique. It makes sense because it is the only think users can distinguish. For direct access see the [IFC documentation](http://www.buildingsmart-tech.org/ifc/IFC4/Add2/html/) and [our examples](http://docs.xbim.net/examples/basic-model-operations.html) please.
